src/components/demos/write/Demo.js

```diff
@@ -27,10 +30,44 @@ const IconWrapper = styled.div`
   text-align: center;
 `
 
+const ConsoleButtonWrapper = styled.div`
+  position: absolute;
+  bottom: 100px;
+  right: 15%;
+  width: 100px;
+  overflow-wrap: break-word;
+  pointer: cursor;
+  text-align: center;
+`
+
 const successfulLogin = (apiOutput) =>
   extractLoginResult(apiOutput) === 'Login succeeded'
 const loginError = (apiOutput) => extractLoginResult(apiOutput) === 'error'
 
+const ConsoleButton = ({ update, state }) => {
```

Comment:
ConsoleButton could be its own component file?

Reply:
It can, ideally wanted to create like a 'IconButton' button so we cann use erverywhere. thant includes hacker app, desktop items annd etc.

Reply:
extrated both into a new file